script/ddgc_upload_metadata.sh

```diff
@@ -25,8 +26,14 @@ NEW_MODIFIED_TIME=0
   NEW_MODIFIED_TIME=$(/usr/bin/stat --format=%Y $DDGC_REPO_JSON_OUT)
 
 if [[ ( $DOWNLOAD_EXIT_CODE -eq 0 && $NEW_MODIFIED_TIME -gt $LAST_MODIFIED_TIME ) || ( $DOWNLOAD_EXIT_CODE -eq 0 && $FORCE_UPLOAD -eq 1 ) ]] ; then
-  bzip2 --best --force --keep $DDGC_REPO_JSON_OUT
+  json_xs < $DDGC_REPO_JSON_OUT > /dev/null
```

Comment:
does this exist in the $PATH?  I mean I'm guessing it does. I just wanted to double check that.